pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/documentation/CommentContentRule.java

```diff
@@ -67,6 +67,10 @@ public void start(RuleContext ctx) {
         if (caseSensitive) {
             currentBadWords = originalBadWords;
         } else {
+            // TODO this only accounts for the original word
+            // + the all uppercase version
+            // but not eg a capitalized version
+            // (hardly a case insensitive comparison)
```

Comment:
It should work, since the search text (commentText) is upper-cased as well later on (if case-insensitive is enabled):
https://github.com/pmd/pmd/pull/1501/files#diff-8814010eee468959e8da8baa38bc02c7R102

Reply:
Oh ok, I hadn't read on that far ^^